### DISCUSSION TITLE
Regarding web3 cli

### DISCUSSION THREAD

**Discussion by anithakc6** [gochain-io/docs#128] [Q&A]
Need steps to install web3 cli using visual studio, in windows 10?
The curl -LSs https://raw.githubusercontent.com/gochain/web3/master/install.sh | sh is giving me the error as

Error: this installer needs the ability to run commands as root.
We are unable to find either "sudo" or "su" available to make this happen.

Can we run this commands in Windows 10, visual studio bash shell?

**Comment by treeder** [comment]
You'll probably have to build `web3` yourself for windows. Or use Windows Linux subsystem.
